daemon/src/org/eclipse/iofog/local_api/CommandLineApiHandler.java

```diff
@@ -50,6 +51,14 @@ public FullHttpResponse call() throws Exception {
 			return new DefaultFullHttpResponse(HTTP_1_1, HttpResponseStatus.METHOD_NOT_ALLOWED);
 		}
 
+		// for Client's "isAnotherInstanceRunning" method
```

Comment:
maybe it's better to reuse the same code for both cases when content type is empty and content type not equal to application/json